lib/LedgerSMB/PSGI.pm

```diff
@@ -14,6 +14,10 @@ LedgerSMB::PSGI - PSGI application routines for LedgerSMB
 use strict;
 use warnings;
 
+use PGObject;
+
+BEGIN { PGObject->new_registry('ledgersmb') }
```

Comment:
Would the new_registry not be a nice argument to the 'use PGObject' call? Seeing `BEGIN` blocks triggers questions :-)